spec-main/api-menu-item-spec.ts

```diff
@@ -5,6 +5,40 @@ import { closeAllWindows } from './window-helpers';
 
 
 describe('MenuItems', () => {
+  describe('MenuItem instance properties', () => {
+    it('should have default MenuItem properties', () => {
+      const item = new MenuItem({
+        id: '1',
+        label: 'hello',
+        role: 'close',
+        sublabel: 'goodbye',
+        accelerator: 'CmdOrControl+Q',
+        click: () => { },
+        enabled: true,
+        visible: true,
+        checked: false,
+        type: 'normal',
+        registerAccelerator: true,
+        submenu: [{ role: 'about' }]
+      })
+
+      expect(item).to.have.property('id').that.is.a('string')
+      expect(item).to.have.property('label').that.is.a('string').equal('hello')
+      expect(item).to.have.property('sublabel').that.is.a('string').equal('goodbye')
+      expect(item).to.have.property('accelerator').that.is.a('string').equal('CmdOrControl+Q')
+      expect(item).to.have.property('click').that.is.a('function')
+      expect(item).to.have.property('enabled').that.is.a('boolean').and.is.true
```

Comment:
without `()` the `true` assertion does nothing (try changing this to `.false` and you'll observe the test still passes).